sdk/keyvault/test-resources.json

```diff
@@ -67,24 +67,24 @@
     },
     "keyVaultSku": {
       "type": "string",
-      "defaultValue": "Premium",
+      "defaultValue": "premium",
```

Comment:
Why lower case is necessary?

Reply:
The deploy resource exception occurred in all cloud tests before fixed.